index.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
 module.exports = {
-    Services: require('./lib/services')
+    Services: require('./lib/services'),
```

Comment:
either both camel or pascal case

Reply:
I initially thought about it. But the reason is that ./lib/services.js exports a class `Services` (classes should be pascal case), while './cli/services.js' is not a class, it just exports a bunch of functions.